submit.go

```diff
@@ -99,7 +99,7 @@ type issueBodyTemplatePayload struct {
 type genericWebhookPayload struct {
 	payload
 	// If a github/gitlab report is generated, this is set.
-	ReportURL  string `json:"report_url"`
+	ReportURL string `json:"report_url"`
```

Comment:
non-blocking comment: any idea why this is happening? Has `gofmt` changed the rules about struct formatting?I'm not convinced that the updated style is an improvement, but I don't care that much as long as we're not just going to flip-flop between the two.

Reply:
Unsure why this is happening tbh. Empirically, this is what `go fmt .` does, so it's not an IDE thing. 
```
go version
go version go1.22.1 darwin/arm64
```

Reply:
same for me. weird.

Reply:
https://tip.golang.org/doc/go1.19#go-doc are you on an old version?

Reply:
No, I'm saying that I'm on go1.22, and it does the same as you. It's entirely possible the current formatting came from an earlier version of go. Thanks for hunting down the changes.